common/djangoapps/student/forms.py

```diff
@@ -275,9 +275,15 @@ def __init__(
                         }
                     )
 
+        extended_profile_field_options = configuration_helpers.get_value('EXTRA_FIELD_OPTIONS', [])
```

Comment:
EXTRA_FIELD_OPTIONS, What do you think of changing the name to: EXTENDED_PROFILE_FIELDS_OPTIONS? @andrey-canon

Reply:
Have you changed the variable name? @andrey-canon